bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/types/BStreamType.java

```diff
@@ -70,7 +70,8 @@ public String toString() {
         if (constraint == BTypes.typeAny) {
             return super.toString();
         } else {
-            return "stream" + "<" + constraint.getName() + ">";
+            return super.toString() + "<" + constraint + ((TypeConstants.ERROR == null) ? 
```

Comment:
You don't need this change unless you can add the Error field to the class and populate it.